### DISCUSSION TITLE
BT-778 (Framework Maximum Participants) is forgotten in the SDK?

### DISCUSSION THREAD

**Discussion by sorliden** [OP-TED/eForms-SDK#74] [Q&A] [answered]
BT-767 (Electronic Auction) and BT-778 (Framework Maximum Participants) are both indicators. If you search for "BT-767" in the SDK files you get plenty of hits. But if you search for "BT-778" you find nothing.
Has BT-778 been forgotten in the SDK?

**Comment by troa1** [comment]
Maybe there are further errors in this one... it does not make sense to have BT-778 as on indicator.... it has to be a number. Indeed, I can find "Framework Maximum Participants Number" as BT-113.

**Reply by sorliden** [reply]
BT-778 is defined as an indicator in the "CELEX table". 
But maybe it's seen as part of BT-113 an there for obsolete.

**Reply by troa1** [reply]
Indeed. However as it is in the annex to the implementing act (CELEX) I would like to see the act changed (remove BT-778) as it is obsolite. Maybe even within 2022 (in the foreseen amendment by November 2022)

**Comment by jpmckinney** [comment, marked as answer]
This is covered by footnote 21 https://docs.ted.europa.eu/eforms/0.7.0/schema/all-in-one.html#_footnotedef_21

> the possible maximum number of participants (BT-113)[[21](https://docs.ted.europa.eu/eforms/0.7.0/schema/all-in-one.html#_footnotedef_21)],
>
> ...
>
> [21](https://docs.ted.europa.eu/eforms/0.7.0/schema/all-in-one.html#_footnoteref_21). Its specification implies a "true" value for BT-778.

**Comment by mgakt** [comment]
Dear All,

As we see, the lack of BT-778 from the SDK implicates incoherence with the law. Based on 1780/2019, this filed is mandatory in notices 16 and 18. Based on developer documentation, this indicator drives the obligatoriness of BT-113. (By 1780/2019, BT-113 is also mandatory.)
So, BT-778 must return to the SDK as soon as possible, or the legal side must be updated.  

P.S.: I checked the previous links in this conversation, but those need to be updated.

**Reply by AnnaKSZ** [reply]
Hello, 
Until the answer comes, check this out
https://docs.ted.europa.eu/eforms/latest/schema/identifiers.html#pointlessDueToDesignSection
